### ISSUE TITLE
beemo fails while bootstrapping lerna

### ISSUE THREAD

**Issue by alvisetrevisan** [williaster/data-ui#162]
Cannot set up the project: after a fresh clone of the repo (master, release 0.75) and `npm install`, during `lerna bootstrap` I get the following error:
```
> @data-ui/data-table@0.0.74 prepublish /Users/a.trevisan/projects/data-ui/packages/data-table
> yarn run build

yarn run v1.13.0
$ yarn run build:cjs && yarn run build:esm
$ beemo babel ./src --out-dir lib/ --react --minify
beemo babel

Transpile files with Babel

Options:
  --version      Show version number                                   [boolean]
  --config       Path to a configuration file             [string] [default: ""]
  --debug        Show debug messages                  [boolean] [default: false]
  --locale       Locale to display output in              [string] [default: ""]
  --output       Control output size (1-3)                 [number] [default: 3]
  --silent       Hide all output                      [boolean] [default: false]
  --theme        Change output colors              [string] [default: "default"]
  --help         Show help                                             [boolean]
  --clean        Clean the target folder              [boolean] [default: false]
  --concurrency  Number of builds to run in parallel                    [number]
  --priority     Prioritize workspace builds based on dependency graph [boolean]
  --workspaces   Run command in each workspace (supports regex)         [string]

Error: Unknown fields: driver.
    at buildAndCheck (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/optimal/lib/optimal.js:48:13)
    at Object.optimal [as default] (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/optimal/lib/optimal.js:68:10)
    at new Routine (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/@boost/core/lib/Routine.js:40:41)
    at new CreateConfigRoutine (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/@beemo/core/lib/configure/CreateConfigRoutine.js:25:1)
    at names.reverse.map.driver (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/@beemo/core/lib/ConfigureRoutine.js:29:29)
    at Array.map (<anonymous>)
    at ConfigureRoutine.setupConfigFiles (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/@beemo/core/lib/ConfigureRoutine.js:28:59)
    at ConfigureRoutine.bootstrap (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/@beemo/core/lib/ConfigureRoutine.js:16:14)
    at ConfigureRoutine.configure (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/@boost/core/lib/Routine.js:67:14)
    at Pipeline.pipe (/Users/a.trevisan/projects/data-ui/packages/data-table/node_modules/@boost/core/lib/Routine.js:122:40)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna info lifecycle @data-ui/data-table@0.0.74~prepublish: Failed to exec prepublish script
```

**Comment by williaster** [comment]
Thanks for raising this issue, we just encountered this in another repo, will fix it asap 👍

**Comment by williaster** [comment]
@ABaldwinHunter should be fixed by #163 if you pull master
